# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/reftests/css-gradients/reftest.list
# Commit: fc2b3d6448bc
# Full Hash: fc2b3d6448bcc3575fffb1abc923e7e703aeef54
# Author: Glenn Watson <git@intuitionlibrary.com>
# Date: 2022-02-24 03:51:41
# Regressor Bug: 1749380
# File Overlap Count: 1
# Description:
#   Bug 1749380 - Part 2 - Performance and quality fixes for part 1. r=gfx-reviewers,nical
#   
#   * Add support for local scale factors to a surface, allowing it to
#     be rasterized in root coordinate space. This allows snapping to
#     work across surfaces where the surface transform is a fractional
# ==============================================================================

diff -r 8c70bc92ad36 -r fc2b3d6448bc layout/reftests/css-gradients/reftest.list
--- a/layout/reftests/css-gradients/reftest.list	Wed Feb 23 20:49:26 2022 +0000
+++ b/layout/reftests/css-gradients/reftest.list	Wed Feb 23 20:49:27 2022 +0000
@@ -55,7 +55,7 @@
 fuzzy-if(Android,0-4,0-248) == radial-zero-length-1d.html radial-zero-length-1-ref.html
 fuzzy-if(Android,0-4,0-248) == radial-zero-length-1e.html radial-zero-length-1-ref.html
 fuzzy-if(Android,0-4,0-248) == radial-zero-length-1f.html radial-zero-length-1-ref.html
-fuzzy(0-1,0-25) == radial-premul.html radial-premul-ref.html
+fuzzy(0-1,0-40) == radial-premul.html radial-premul-ref.html
 == repeated-final-stop-1.html repeated-final-stop-1-ref.html
 == repeating-linear-1a.html repeating-linear-1-ref.html
 == repeating-linear-1b.html repeating-linear-1-ref.html